frontend/src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -92,6 +92,9 @@ export const FeatureToggleListTable: VFC = () => {
         'features-list-table',
         stateConfig,
     );
+    // offset needs to be first so we can override it
+    const setTableStateWithOffsetReset: typeof setTableState = (data) =>
```

Comment:
this is a new method we use instead of the low level setTableState to never forget about the offset reset

Reply:
Can we define this method somewhere near state, so we do not have to rewrite it in every component, but just take it from our hook.

Reply:
we need config definition for this to work and autocomplete the offset field